src/frontend/app.ts

```diff
@@ -1,343 +0,0 @@
-import bodyParser from "body-parser";
```

Comment:
This file used to be the top level router, which is now multiple hierarchical routers in src/routes